src/AspNetCoreModuleV2/InProcessRequestHandler/StartupExceptionApplication.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#include "StartupExceptionApplication.h"
+
+VOID StartupExceptionApplication::ShutDown()
+{
+    return VOID();
+}
+
+VOID StartupExceptionApplication::Recycle()
+{
+    return VOID();
```

Comment:
Are you sure this should be empty?

Reply:
Good point. Let me think about it.

Reply:
I am sure about the value of this change. We introduce multiple classes just for handling one error page. Why not just a simple UTILITY method?

Reply:
If we want to update the error page in the future, we would need to modify the shim. If we want to use a utility function, we will effectively reuse the inprocess application loaded to just return 500s.